teamd/teamd_runner_ttdp.c

```diff
@@ -2296,6 +2367,13 @@ static int ab_init(struct teamd_context *ctx, void *priv)
 		teamd_log_err("Failed to register state value group.");
 		goto event_watch_unregister;
 	}
+	for (int i = 0; i < 4; i++) {
```

Comment:
Maybe use a constant instead of magic number `4`, see comment in [`teamd/teamd_lw_ttdp.h`](https://github.com/westermo/libteam/pull/1/files#r299431030).

Reply:
Fixed!